src/model/package.ts

```diff
@@ -17,6 +17,18 @@ import {Document, FeatureKinds} from './document';
 import {Feature} from './feature';
 import {Queryable} from './queryable';
 
+export type QueryOptions = object & {
+  /**
+   * If true then results will include features from outside the package, e.g.
+   * from files in bower_components or node_modules directories.
+   */
+  includeExternalCode?: boolean;
```

Comment:
I'd suggest `includeExternalLibs`, `externalLibs`, `includePackages`, or just `packages`. It'd be nice to align this with the `imports` option name, even if they're not now used together, or never will.